Changelog.md

```diff
@@ -393,14 +393,14 @@ concurrently using this.
   stream processing function application pipeline concurrently.
 * Added `mapMaybe` and `mapMaybeM`.
 
-## 0.2.1
+## 0.2.1 - 5th June 2018
 
 ### Bug Fixes
 * Fixed a bug that caused some transformation ops to return incorrect results
   when used with concurrent streams. The affected ops are `take`, `filter`,
   `takeWhile`, `drop`, `dropWhile`, and `reverse`.
 
-## 0.2.0
+## 0.2.0 - 13th May 2018
```

Comment:
14th May according to https://github.com/composewell/streamly/releases

Reply:
I used Hackage upload dates which are likely UTC so a few hours behind